__device-tests__/pages/editor-page.js

```diff
@@ -72,6 +72,13 @@ export default class EditorPage {
 		return await this.driver.elementByXPath( blockLocator );
 	}
 
+	async getTitleElement() {
```

Comment:
Hey, @JavonDavis @etoledom I have resolved merge conflicts and I need to help from someone of you to check if everything is on the place in this file :) Thanks! :)

Reply:
`getTitleElement` seems to be duplicated. That's probably why the UI Tests are failing.
@JavonDavis might have a better context to point out which is the correct one. (Probably the one already on dev?)

Reply:
@etoledom you're right here, the one already in dev is the one we should keep, this duplicate function can be removed

Reply:
Thanks @etoledom @JavonDavis , removed!